src/transaction/boot_sr.h

```diff
@@ -103,7 +103,6 @@ extern VOLID boot_find_next_permanent_volid (THREAD_ENTRY * thread_p);
 extern int boot_reset_db_parm (THREAD_ENTRY * thread_p);
 extern DKNPAGES boot_max_pages_new_volume (void);
 extern DKNPAGES boot_max_pages_for_new_auto_volume_extension (void);
-extern DKNPAGES boot_max_pages_for_new_temp_volume (void);
 extern DKNPAGES boot_get_temp_temp_vol_max_npages (void);	/* todo: remove me */
```

Comment:
The update looks very good to me. 

One observation I found is `boot_get_temp_temp_vol_max_npages` is also unused.
Would you also remove it?